manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -82,22 +82,47 @@ spec:
     rules:
     - alert: UsingDeprecatedAPIAppsV1Beta1
       annotations:
-        message: A client in the cluster is using deprecated apps/v1beta1 API that will be removed soon.
+        summary: "A client is using a deprecated {{"{{"}} $labels.group {{"}}"}}/{{"{{"}} $labels.version {{"}}"}} API version"
+        message: |
+          A client in the cluster has used a deprecated {{"{{"}} $labels.group {{"}}"}}/{{"{{"}} $labels.version {{"}}"}} API version of the {{"{{"}} $labels.resource {{"}}"}} resource in the last 24 hours.
+          This API version will be removed soon; please migrate to the current version of this API.
       expr: |
-        apiserver_request_count{group="apps",version="v1beta1"}
+        increase(
+          apiserver_request_total{
+            group="apps",
+            version="v1beta1",
+          }[24h]
+        ) > 0
       labels:
         severity: warning
     - alert: UsingDeprecatedAPIAppsV1Beta2
       annotations:
-        message: A client in the cluster is using deprecated apps/v1beta2 API that will be removed soon.
+        summary: "A client is using a deprecated {{"{{"}} $labels.group {{"}}"}}/{{"{{"}} $labels.version {{"}}"}} API version"
+        message: |
+          A client in the cluster has used a deprecated {{"{{"}} $labels.group {{"}}"}}/{{"{{"}} $labels.version {{"}}"}} API version of the {{"{{"}} $labels.resource {{"}}"}} resource in the last 24 hours.
+          This API version will be removed soon; please migrate to the current version of this API.
       expr: |
-        apiserver_request_count{group="apps",version="v1beta2"}
+        increase(
+          apiserver_request_total{
+            group="apps",
+            version="v1beta2",
+          }[24h]
+        ) > 0
       labels:
         severity: warning
     - alert: UsingDeprecatedAPIExtensionsV1Beta1
       annotations:
-        message: A client in the cluster is using deprecated extensions/v1beta1 API that will be removed soon.
+        summary: "A client is using a deprecated {{"{{"}} $labels.group {{"}}"}}/{{"{{"}} $labels.version {{"}}"}} API version"
+        message: |
+          A client in the cluster has used a deprecated {{"{{"}} $labels.group {{"}}"}}/{{"{{"}} $labels.version {{"}}"}} API version of the {{"{{"}} $labels.resource {{"}}"}} resource in the last 24 hours.
+          This API version will be removed soon; please migrate to the current version of this API.
       expr: |
-        apiserver_request_count{group="extensions",version="v1beta1",resource!~"ingresses|",client!~"hyperkube/.*|cluster-policy-controller/.*"}
+        increase(
+          apiserver_request_total{
+            group="extensions",
+            version="v1beta1",
+            client!~"hyperkube/.*|cluster-policy-controller/.*|velero-server/.*"
+          }[24h]
+        ) > 0
       labels:
         severity: warning
```

Comment:
severity level is hasn't changed (should be info, not warning)

Reply:
Nope, that won't change as stated before.
/hold cancel

Reply:
@soltysh I'm confused as here (https://github.com/openshift/cluster-kube-apiserver-operator/pull/730#pullrequestreview-348095535) you stated that "info" was okay, and I had updated my original PR to reflect this.

Warning alerts should be indicative of the cluster health being degraded in some way, and action is required. In the case of OSD, warning alerts are sent to SREs. This alert is **not** actionable in this way, and should not be at a warning level. It's a bad experience for RH SREs, and both OSD and OCP customers as it's a piece of information that they should take action before the next minor version upgrade.

Reply:
Seems like info would work just fine for all known use-cases.  Can we go with that, please?

Reply:
Note that regardless of the severity level it will still show up in the main console view.

Also:
> This alert is not actionable in this way, and should not be at a warning level. 

Exactly why this is why I think we should reconsider this as it can induce alert fatigue, as it will just be there. Are these all components users can control and migrate away from this client?

Reply:
@soltysh pinging again here - request to move to info level.

Reply:
@soltysh can you please set the severity of these alerts to info rather than warning?

Reply:
Should this now change to info, to be in line with the master PR (https://github.com/openshift/cluster-kube-apiserver-operator/pull/742/files#diff-9cc7eba97b17dc12b44afb453dc9e69cL126) ?

Reply:
The master PR is about different resource that is still present in Kubernetes.

These ones are already removed in the Kubernetes version we are shipping and we'll drop the hack reenabling them in the next release - so warning seems pretty warranted to me. There should be action taken to hunt down the team using it. I think most of our customer use the clusters internally and care about apps that will stop working after the next upgrade. There is also the other case of clusters like Online where you provision for outside users and don't care about what they run, so I'd just suppress the particular warning there unless there is a better way.